docs/decisions/0007-lti-1p3-decoupling.rst

```diff
@@ -0,0 +1,115 @@
+0007 Decouple LTI 1.3 Launch from XBlock and edX Platform
+#########################################################
+
+Status
+******
+**Draft (=> Accepted)**
+
+Context
+*******
+
+In this document, the terms context and launch information are important. Context describes the technological
```

Comment:
When you used words `context` and `Launch Information`, I believe these are code variable names, right? Can you make that explicit? I confused `context` with the title of this section.